samples/azure-quantum/entanglement-swap/ES-quantinuum-qsharp.ipynb

```diff
@@ -0,0 +1,1120 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Quantum Teleportation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "$\\providecommand{\\ket}[1]{\\left|#1\\right\\rangle}$\n",
+    "$\\providecommand{\\bra}[1]{\\left\\langle#1\\right|}$\n",
+    "\n",
+    "In this sample we will be looking at using quantum teleportation on Azure Quantum.\n",
+    "\n",
+    "To refresh our mind let us review how quantum teleportation works.\n",
+    "\n",
+    "We have two parties, Alice and Bob, they would like to share information, and they have an entangled state (usually the Bell state $\\frac{1}{\\sqrt{2}} (\\ket{00} + \\ket{11})$). \n",
+    "If Alice now wishes to transfer quantum information, she will entangle some payload with her qubit and measure her qubit and payload.She will transmit the results of the measurement to Bob, who can use them to reconstruct Alice's payload on his entangled qubit.\n",
+    "\n",
+    "In the following we code will implement this operation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/x-qsharp-data": "[]",
+      "text/html": [
+       "<ul></ul>"
+      ],
+      "text/plain": []
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "open Microsoft.Quantum.Diagnostics;"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/x-qsharp-data": "[\"CreateBellState\",\"Send\",\"Receive\"]",
+      "text/html": [
+       "<ul><li>CreateBellState</li><li>Send</li><li>Receive</li></ul>"
+      ],
+      "text/plain": [
+       "CreateBellState, Send, Receive"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "operation CreateBellState(alice : Qubit, bob : Qubit) : Unit is Adj + Ctl {\n",
```

Comment:
Nit, but the typical verb in Q# is "prepare" rather than "create." In particular, "create" may carry the implication of making or allocating new qubits, while "prepare" emphasizes that the operation acts on the qubits that it is given.